pino.js

```diff
@@ -58,6 +58,8 @@ function pino (opts, stream) {
     buf: ''
   }
   iopts.chindings = ''
+  iopts.levelMap = Object(null)
```

Comment:
@mcollina or @davidmarkclements might disagree, but I'd rather see `iopts.levelMap = {}`. I do not like the broken object returned by `Object(null)` (which, in my opinion, should be `Object.create(null)` anyway).